11/collection-meta.yaml

```
@@ -128,6 +128,11 @@ collections:
     repository: https://github.com/ansible-collections/community.mysql
   community.network:
     repository: https://github.com/ansible-collections/community.network
+    removal:
+      major_version: 12
+      announce_version: 11.0.0a2
```

Assuming there is no further alpha release, this needs to be updated to:
```suggestion
      announce_version: 11.0.0b1
```